comparisons/CompareCrossSecToFONLL.py

```diff
@@ -78,6 +90,10 @@
     hFONLLPromptCentral.Scale(1.e-6 / BR / 20, 'width')
     hFONLLPromptMin.Scale(1.e-6 / BR / 20, 'width')
     hFONLLPromptMax.Scale(1.e-6 / BR / 20, 'width')
+    for iPt in range(hFONLLPromptCentral.GetNbinsX()):
+        hFONLLPromptCentral.SetBinError(iPt+1, 1.e-20)
+        hFONLLPromptCentral.SetBinError(iPt+1, 1.e-20)
+        hFONLLPromptCentral.SetBinError(iPt+1, 1.e-20)
```

Comment:
Here there is a copy-paste error I think. If the uncertainty is needed to plot the points without the "line" joining them, this should already be corrected in the new prediction files that I uploaded and should not be needed with the "old" files.

Reply:
Ciao @fcatalan92, indeed there is a copy-paste mistake for the three histos (moreover the central one only should be fine) and I missed the last update of the input files. I now removed them, thanks!